src/test/java/test/jackson/JsonNodeAssert.java

```diff
@@ -0,0 +1,36 @@
+package test.jackson;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.assertj.core.api.AbstractAssert;
+
+import java.util.Objects;
+
+import static com.fasterxml.jackson.databind.node.JsonNodeType.STRING;
+
+/**
+ * A simple custom AssertJ assertion class for handling basic {@link JsonNode} assertions.
+ */
+public class JsonNodeAssert extends AbstractAssert<JsonNodeAssert, JsonNode> {
```

Comment:
Not strictly necessary, but gives us cleaner assertions in the tests. Quite surprised such a thing doesn't already exist.